djangoplicity/pages/tests/test_models.py

```diff
@@ -0,0 +1,150 @@
+from django.conf import settings
+from datetime import datetime, timedelta
+from django.test import TestCase
+from factories import SectionFactory, PageFactory, URLFactory, PageGroupFactory, EmbeddedPageKeyFactory
+from djangoplicity.pages.models import (
+    Page, URL, EmbeddedPageKey, Section,
+    register_page_key, generate_page_id, page_cache_invalidator, 
+    url_cache_invalidator, page_key_cache_invalidator, section_cache_invalidator
+    )
+
+from mock import patch
+
+class SectionTest(TestCase):
+
+    def test_unicode_representation(self):
+        section = SectionFactory.build()
+        self.assertEqual(u'%s' % section, 'TestingSection')
```

Comment:
Why are you using unicode strings in here?. Doesn't this work?
```suggestion
        self.assertEqual(section, 'TestingSection')
```

Reply:
Resolved in [#ad182f52](https://github.com/djangoplicity/djangoplicity/pull/96/commits/ad182f52f0a9dbcfe1655e63e109986108265504)